playbook/app/pb_kits/playbook/pb_table/table_header.rb

```diff
@@ -11,7 +11,7 @@ class TableHeader < Playbook::KitBase
                            default: "center"
       prop :justify_sort_icon, type: Playbook::Props::Enum,
                                values: %w[start center end stretch around between evenly none],
-                               default: "between"
+                               default: "none"
```

Comment:
I don't think this should be changing the default behavior because the UI changes were just thought up during planning and not by Design/Jason